src/callbacks.rs

```diff
@@ -95,4 +95,9 @@ pub trait ParseCallbacks: fmt::Debug + UnwindSafe {
     ) -> Option<ImplementsTrait> {
         None
     }
+
+    /// Provide a list of custom derive attributes.
+    fn add_derives(&self, _name: &str) -> Option<Vec<String>> {
```

Comment:
Let's change this to just return a `Vec<String>`, and default `vec![]`? Or is there any reason we should distinguish between `None` and `Some(vec![])`?

Reply:
No reason, that change is fine with me.